ansible_bender/core.py

```diff
@@ -173,6 +174,29 @@ def _get_path_our_site(self):
         # hence, let's add the site ab is installed in to sys.path
         return os.path.dirname(os.path.dirname(ansible_bender.__file__))
 
+    def _correct_host_entries(self, playbook_path, tmpDir):
+        """ Correct the host entries in the playbook and all imported playbooks """
+        tmp_pb_path = os.path.join(tmpDir, str(uuid.uuid4()) + ".yaml")
```

Comment:
love using the uuid here! could we please prefix the uuid string with e.g. `ab_` so that people know the file is coming from ansible-bender?